sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/plans/logical/v2Commands.scala

```diff
@@ -241,10 +241,11 @@ case class ReplaceTableAsSelect(
  * The logical plan of the CREATE NAMESPACE command that works for v2 catalogs.
  */
 case class CreateNamespace(
-    catalog: SupportsNamespaces,
-    namespace: Seq[String],
+    child: LogicalPlan,
```

Comment:
can we delay this change? We may want to create the rule to convert to v1 commands when migrating to the new framework. Let's focus on disallowing reserved properties in this PR.

Reply:
OK